### ISSUE TITLE
question: Does dragula support any functionality to move mirrors to cursor position?

### ISSUE THREAD

**Issue by eLeontev** [bevacqua/dragula#616]
Hi, unfortunately I did not see the answer, so I asked it here:
the case is next:
<div class='entity'></div>
<style>
.entity {
  width: 100px
}
.gu-mirror.entity {
 width: 20px;
}
</style>

In this case the mirror will be left of the cursor if an entity will be drag in the right part of itself.

The goal is to move mirror to the cursor (middle of the mirror on cursor)

If the library does not support such case, is it correct to use `transfrom: translate(calculatedOffest(event))` ?

**Comment by thekikao** [comment]
+1

**Comment by brandoncc** [comment]
I agree, I wish this existed. Since the repo isn't maintained, I might fork and take a stab at it.

**Comment by Metroslim** [comment]
Hi there, I've made a PR for that. Adding an option to place the mirror item under the pointer and let you play with css to make it 50% 50% with transform or whatever relative position you like.
